### DISCUSSION TITLE
Reducing timing between packets transformation from ~10ms to ~1ms

### DISCUSSION THREAD

**Discussion by TheAvengerPen** [Haivision/srt#2025] [Q&A] [answered]
Hi,

I am trying to set up SRT under a GRE tunnel in Linux Ubuntu 20.04. I am currently using SRT v1.4.1. From what I have read in issues such as [#637](https://github.com/Haivision/srt/issues/637) and [#673](https://github.com/Haivision/srt/issues/673) where they seem to have the same issue. 

However, I am struggling to follow what is being said. One such place is the reference to the NO_BUSY_WAITING definition which should be removed from srt.h and udt.h; however, this definition does not seem to be in either srt.h or udt.h. Is this something that has been moved/changed in later iteration of SRT as I realise these questions were posed in 2019.

Is the idea still the same throughout or have you updated anything to change the sleep_until() usage.

Any help will be very much appreciated.

**Comment by maxsharabayko** [comment, marked as answer]
There is a CMake build option `-DUSE_BUSY_WAITING=ON` to enable a CPU-busy sending loop.
